syntax = "proto3";
package YL.Base;

//服务ID
enum ServiceID {
    SID_ZERO            = 0x0000;   //
    SID_LOGIN           = 0x0001;   //登录
    SID_FRIEND_LIST     = 0x0002;  //好友列表
    SID_MSG             = 0x0003;   //好友消息
    SID_GROUP           = 0x0004;   //群组消息
    SID_OTHER		= 0x0005;
}


enum LoginCmdID{
    CID_ZERO                   = 0x0000;
    //登录消息服务器
    CID_LOGIN_REQ_MSGSERVER    = 0x0101;
    CID_LOGIN_RES_MSGSERVER    = 0x0102;
    //用户登录
    CID_LOGIN_REQ_USERLOGIN    = 0x0103;
    CID_LOGIN_RES_USERLOGIN    = 0x0104;
    //用户注销
    CID_LOGIN_REQ_USERLOGINOUT = 0x0105;
    CID_LOGIN_RES_USERLOGINOUT = 0x0106;
}


enum ResultType {
    REFUSE_REASON_NONE              = 0;
    REFUSE_REASON_NO_MSG_SERVER     = 1;
    REFUSE_REASON_MSG_SERVER_FULL   = 2;
}
